packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -345,6 +345,19 @@ class DDSMastheadComposite extends HostListenerMixin(LitElement) {
         `;
       }
 
+      if (elem.isHeading) {
```

Comment:
Do we need this check, or can we add `.isHeading="${elem.isHeading}"` to the returned template fragment that follows this conditional instead?